cmd/mdatagen/main.go

```diff
@@ -320,6 +320,17 @@ func templatize(tmplFile string, md metadata) *template.Template {
 				"expectConsumerError": func() bool {
 					return md.Tests.ExpectConsumerError
 				},
+				"supportsWindows": func() bool {
+					if len(md.Status.SupportedPlatforms) == 0 {
+						return true
+					}
+					for _, platform := range md.Status.SupportedPlatforms {
+						if platform == "windows" {
+							return true
+						}
+					}
+					return false
+				},
```

Comment:
This is a too specific a solution for our use case. Let's start with `unsupported_platforms` option in metadata.yaml and just list them in `go:build` pragma